app/views/retro/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="nav">
+  <div class="headerItems">
+    <Button id="start-new-button" class="start-button">Post your topics!</Button>
+    <Button id="time"></Button>
+    <%= image_tag("/assets/wye-logo.png", size: "40x40", class: "header-icon") %>
+  </div>
+</div>
```

Comment:
I think the header will be throughout the whole application, so what if you write it in a partial and render it in each view so you do not have to put this code in every view you write?

If there's a reason to do it this way, I'm fine.

Reply:
When I wrote the header (and still), I didn't really know if the header would be everywhere and I since is not specified in any AC I didn't really thought about making it a more generic component with a partial view.
But I agree that it would be a small change and could be useful for future pages :)